gulp-tasks/utils/build-browser-bundle.js

```diff
@@ -50,7 +52,16 @@ const globals = (moduleId) => {
   const packagePath = path.join(__dirname, '..', '..', 'packages', packageName);
   try {
     const pkg = require(path.join(packagePath, 'package.json'));
-    return `${constants.NAMESPACE_PREFIX}.${pkg.workbox.browserNamespace}`;
+    const pkgNamespace = pkg.workbox.browserNamespace;
```

Comment:
I'm not opposed to any of these changes but it feels seperate from the PR you are trying to solve.

Reply:
Yep, removed the comments and reverted to the original version. This was just experimentation I forgot about.